projects/Mallard/src/components/front/items/super-items.tsx

```diff
@@ -97,8 +99,8 @@ const SportSuper = ({ article, size, ...tappableProps }: PropTypes) => {
             />
             <View
                 style={[
-                    sportSuperStyles.card,
                     superHeroImageStyles.textStandBlock,
```

Comment:
this is pointless as a change (i was trying things) but it's a more coherent specificity order so i might as well keep it